packages/mobile/src/screens/profile-screen/ProfileTabNavigator.tsx

```diff
@@ -48,7 +49,7 @@ export const ProfileTabNavigator = ({
   const trackScreen = collapsibleTabScreen({
     name: 'Tracks',
     Icon: IconNote,
-    component: TracksTab,
+    component: () => <TracksTab isProfileLoaded={isProfileLoaded} />,
```

Comment:
I believe this leads to perf issues, since this component is not memoized, i think the solution is to provide children to the tab like we did with BottomTabNavigator back in the day. Thinking what we might be able to do is just add the `isProfileLoaded` selector into TracksTab?

Reply:
I was able to set `handle` on initialParams and pull that out in `TracksTab` using `useRoute`. Let me know what you think!

Reply:
yeah that's perf